README.md

```diff
@@ -759,6 +759,8 @@ devices that go beyond what is available in [`embedded-hal`]:
 
 ## Driver crates
 
+You can also find many of these drivers [here](https://tweedegolf.github.io/drive-rs/); please consider adding your driver there (https://github.com/tweedegolf/drive-rs) while we work to port everything across.
```

Comment:
```suggestion
You can also find many of these drivers [on drive-rs](https://tweedegolf.github.io/drive-rs/); please consider adding your driver there ([in the GitHub repo](https://github.com/tweedegolf/drive-rs)) while we work to port everything across.
```

as per [standard accessibility guidelines](https://webaim.org/techniques/hypertext/link_text#uninformative) you shouldn't name links "here" (e.g. important for people with screen readers)

Reply:
oh, good to know, thanks :) also reads much better!